templates/participant/details.html.twig

```diff
@@ -179,7 +179,9 @@
                                             {% for type, label in digitalHealthSharingTypes %}
                                                 <dd>{{ label }} {{ macros.displayDigitalSharing(participant.digitalHealthSharingStatus, type) }}</dd>
                                             {% endfor %}
-                                        </dl>
+                                            </dl>
+                                            <dt>Exploring the Mind Consent</dt>
+                                            <dd>{{ macros.displayConsentPanelStatus(participant, 'EtMConsent', '', true) }}</dd>
```

Comment:
This should be consentForEtM instead of EtMConsent. Also, I confirmed with Gwen we don't need to display red x or green check. Could you pass false instead of true so that we don't display those check marks?